addons/web/static/tests/views/list_tests.js

```diff
@@ -4335,6 +4336,42 @@ QUnit.module('Views', {
         list.destroy();
     });
 
+    QUnit.test('pressing ENTER/TAB in editable list view should keep list in editable mode', async function (assert) {
+        assert.expect(1);
+
+        testUtils.mock.patch(ListController, {
+            _setMode: function (mode, recordID) {
+                if (mode === "readonly") {
+                    throw new Error("should not set mode to readonly");
+                }
```

Comment:
@SimonGenin the code will never reach to error, pressing Enter/TAB in editable listview will create new record and we passed ```stayInEdit``` in options so if anyhow in future if pressing Enter/Tab makes list mode readonly then this test case will fail as it will throw error.

Reply:
you can do the same things with:
```js
assert.notEqual(mode, "readonly", "should not set mode to readonly");
```